docs/management_node.md

```diff
@@ -188,7 +188,9 @@ racker status
 ```
 
 If the BMCs were not reachable, check their MAC addresses and the wireing of the NICs to the internal switch.
-Also try to power-cycle the rack to force them to pick up a new IP address via DHCP.
+In case the internal subnet or the MAC address list was changed you should wait 2 minutes before retrying so that the BMCs had a chance to pick up the new IP addresses via DHCP.
+If there was a previous DHCP configuration with a longer lease time, you can also try to power-cycle the rack to force a DHCP renewal or first switch to the old subnet with `racker bootstrap … -subnet-prefix a.b.c` and then run `ipmi --all lan set 1 ipsrc dhcp` which should trigger a DHCP renewal.
+If IPMI static IP addressing was used you have to switch the BMCs back to DHCP (either manually or by switching to the same subnet with `racker bootstrap … -subnet-prefix a.b.c` and then running `ipmi --all lan set 1 ipsrc dhcp`).
```

Comment:
```suggestion
If IPMI static IP addressing was used during provisioning, you have to switch the BMCs back to DHCP (either manually or by switching to the same subnet with `racker bootstrap … -subnet-prefix a.b.c` and then running `ipmi --all lan set 1 ipsrc dhcp`).
```

Reply:
you can ignore the suggestion and rephrase it as per your linking. Either way, existing sentence is confusing.

Reply:
This is about the IPMI configuration not the node's public IP address which can be static or dynamic, too. For the IPMI IP address we use and assume DHCP but in case some BMC was configured to have a static address, this needs to be switched to DHCP through the IPMI command.

Reply:
there needs to be some context around `was used` so that its not confusing.

Reply:
Thanks, done